plugin/SDKWrapper.php

```diff
@@ -205,4 +188,30 @@ public function getWPOptionWrapper(): WPOptionWrapperBase
 	{
 		return $this->_pWPOptionWrapper;
 	}
+
+
+	/**
+	 * @param array $curlOptions
+	 * @return SDKWrapper
+	 */
+
+	public function withCurlOptions(array $curlOptions): self
+	{
+		$pClone = clone $this;
+		$pClone->_pSDK->setApiCurlOptions($curlOptions);
+		return $pClone;
+	}
+
+
+	/**
+	 *
+	 */
+
+	public function __clone()
+	{
+		if (is_object($this->_pSDK)) {
```

Comment:
how can `$this->_pSDK` not be an object? The variable gets initialized in the constructor...